src/_fetch/_headers.js

```diff
@@ -5,16 +5,27 @@ const makeHeaders = function(options) {
     options['Api-User-Agent'] ||
     'User of the wtf_wikipedia library'
 
+  let origin
+  if (options.noOrigin) {
+    origin = ''
+  } else {
+    origin = 
+      options.origin ||
+      options.Origin ||
+      '*'
+  }
+
```

Comment:
the if statement is fine but a [Conditional (ternary) operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)